core/env.js

```diff
@@ -85,14 +85,21 @@
   
   /** Get a list of boards that we know about */
   function getBoardList(callback) {
-    Espruino.Core.Utils.getJSONURL(JSON_DIR + "boards.json", function(boards){
+    var jsonDir = Espruino.Config.BOARD_JSON_URL || DEFAULT_JSON_DIR;
```

Comment:
Ah, you're right. `DEFAULT_JSON_DIR` is not necessary. I've pushed a commit that removes it.